qcodes/dataset/data_export.py

```diff
@@ -86,30 +87,18 @@ def _get_data_from_ds(ds: DataSet) -> List[List[Dict[str, Union[str, np.ndarray]
     for dep_name, data_dict in parameter_data.items():
         data_dicts_list = []
 
-        dep_data_dict_index = None
-
-        for param_name, data in data_dict.items():
-            my_data_dict: Dict[str, Union[str, np.ndarray]] = {}
-
-            my_data_dict['name'] = param_name
-
-            my_data_dict['data'] = data.flatten()
-
-            ps = ds.paramspecs[param_name]
-            my_data_dict['unit'] = ps.unit
-            my_data_dict['label'] = ps.label
+        dependent = ds.paramspecs[dep_name].base_version()
```

Comment:
inderdependencies class has the __getitem__ implemented, so this should work
```suggestion
        dependent = ds.description.interdeps[dep_name]
```